tueplots/figsizes.py

```diff
@@ -476,6 +476,29 @@ def neurips2023(
     )
 
 
+def neurips2024(
+    *,
+    rel_width=1.0,
+    nrows=1,
+    ncols=2,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """Neurips 2024 figure size.
+    Source: https://nips.cc/Conferences/2024/CallForPapers"""
```

Comment:
Can we introduce a few linebreaks here to make the docstrings more similar to the others? Something like:

```suggestion
    """Neurips 2024 figure size.
    
    Source: https://nips.cc/Conferences/2024/CallForPapers
    """
```